library/src/main/java/co/moonmonkeylabs/realmrecyclerview/RealmRecyclerView.java

```diff
@@ -97,8 +107,13 @@ protected void onMeasure(int widthSpec, int heightSpec) {
     }
 
     private void init(Context context, AttributeSet attrs) {
-        inflate(context, R.layout.realm_recycler_view, this);
         initAttrs(context, attrs);
+        // the layout was hardcoded for vertical orientation: easy way out? create a duplicate but horizontal
```

Comment:
Instead of having the conditional statement here for the layout, you could change that so it's conditional for horizontal and set the scrollbar and layout_height values accordingly.